### PR TITLE
Fix parent and trace ID being mixed up in example

### PR DESCRIPTION
**Pull request by arminru**
and fix some grammar/inconsistencies

### CODE REVIEW

File: spec/20-http_header_format.md [hunk 1]
```diff
@@ -99,11 +99,11 @@ trace-flags      = 2HEXDIGLC   ; 8 bit flags. Currently only one bit is used. Se
 
 #### Trace-id
 
-Is the ID of the whole trace forest. It is represented as a 16-bytes array, for
```
Comment (arminru) [inline spec/20-http_header_format.md:102]: By the way, what is a *trace forest* supposed to be?
> ```#### Trace-id```
```Is the ID of the whole trace forest.```

**Review by discostu105 — APPROVED** [review]

**Comment by discostu105** [comment]
Marked as non substantive for IPR from ash-nazg.

File: spec/20-http_header_format.md (hunk 1 above)
Comment (SergeyKanzhelev) [reply spec/20-http_header_format.md:102]: it is multi-tree concept. We will bring description back https://github.com/w3c/trace-context/issues/279

File: spec/20-http_header_format.md [hunk 2]
```diff
@@ -142,9 +142,9 @@ rather than strict rules to follow for three reasons:
 1. Trust and abuse
 2. Bug in caller
 3. Different load between caller service and callee service might force callee
-   to down sample.
+   to downsample.
 
-You can find more in security section of this specification.
+You can find more in the section *Security considerations* of this specification.
```
Comment (SergeyKanzhelev) [inline spec/20-http_header_format.md]: since you touched it - you can insert the link here

Comment (SergeyKanzhelev) [reply spec/20-http_header_format.md]:
```suggestion
You can find more in the section [Security considerations](#security-considerations) of this specification.
```

**Review by SergeyKanzhelev — APPROVED** [review]
thank you!

**Review by danielkhan — APPROVED** [review]

**Review by danielkhan — APPROVED** [review]

**Pull request merged**